consensus/src/epoch_manager.rs

```diff
@@ -542,6 +534,35 @@ impl EpochManager {
         self.block_retrieval_tx = None;
     }
 
+    async fn start_recovery_manager(
+        &mut self,
+        ledger_data: LedgerRecoveryData,
+        epoch_state: EpochState,
+    ) {
+        let network_sender = NetworkSender::new(
+            self.author,
+            self.network_sender.clone(),
+            self.self_sender.clone(),
+            epoch_state.verifier.clone(),
+        );
+        let (round_manager_tx, round_manager_rx) = aptos_channel::new(
+            QueueStyle::LIFO,
+            1,
+            Some(&counters::ROUND_MANAGER_CHANNEL_MSGS),
```

Comment:
`ROUND_MANAGER_CHANNEL_MSGS` -> `RECOVERY_MANAGER_CHANNEL_MSGS`

Reply:
it shared the same channel as round manager, I don't think it's worth a separate counter